ptf

```diff
@@ -64,7 +64,7 @@ config_default = {
     # Switch connection options
     "platform"           : "eth",
     "platform_args"      : None,
-    "platform_dir"       : None,
+    "platform_dir"       : "/etc/ptf/platforms",
```

Comment:
the rest we can merge, but modifying this is not really ok
if you want to install the `platforms` file there, then you can can use the `--platform-dir` command line option for ptf
otherwise, you should put these files somewhere in the python path